src/EnergyPlus/FluidProperties.cc

```diff
@@ -2779,7 +2770,7 @@ namespace FluidProperties {
         if (!state.dataGlobal->WarmupFlag && ErrorFlag) {
             ++this->errors[(int)RefrigError::SatTemp].count;
             // send warning
-            if (this->errors[(int)RefrigError::SatTemp].count <= df->RefrigErrorLimitTest) {
+            if (this->errors[(int)RefrigError::SatTemp].count <= state.dataFluidProps->RefrigErrorLimitTest) {
```

Comment:
[src/EnergyPlus/FluidProperties.cc:2762]:(style),[variableScope],The scope of the variable 'df' can be reduced.